model/clusters_mgmt/v1/identity_provider_type.model

```diff
@@ -132,6 +132,12 @@ struct HTPasswdIdentityProvider {
 
 	// Password to be used in the _HTPasswd_ data file.
 	Password String
+
+    // Link to the collection of _HTPasswd_ users.
+    link Users []HTPasswdUser
+
+    // Actual collection of _HTPasswd_ users.
+    Items []HTPasswdUser
```

Comment:
This `Items` isn't needed. The `Users` link already contains an inner `items` attribute.